web/src/stories/content/Instruction.js

```diff
@@ -4,7 +4,10 @@ import RenderViewbox from '../RenderViewbox';
 import Instruction from '../../components/Instruction';
 
 class InstructionStory extends Component {
```

Comment:
Is this still a class for the same reasons as the one above (`useEffect` and tests being funky together)?

Reply:
It's because it's in Storybook and so in my calculus the effort to change it wasn't worth nearly as much, so I didn't do it. 😛

Reply:
It's because in the Storybook and I didn't think it was worth fixing, tbh.